src/Entity/AlertBannerEntity.php

```diff
@@ -406,7 +409,21 @@ public function getCacheContexts() {
       $this->addCacheContexts($contexts);
     }
 
-    return parent::getCacheContexts();
+    return Cache::mergeContexts(parent::getCacheContexts(), $contexts);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+
+    $cache_tags = [];
+
+    // Get token and use as a cache tag.
+    $token = $this->getToken();
+    $cache_tags = Cache::mergeTags($cache_tags, ['localgov.alert.banner.token:' . $token]);
```

Comment:
This can be simplified to `$cache_tags = ['localgov.alert.banner.token:' . $token];`